### DISCUSSION TITLE
when i use useRouter

### DISCUSSION THREAD

**Discussion by BADHON512** [vercel/next.js#47637] [Help]
### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

```bash
when i use useRouter
```

### Which example does this report relate to?

Error: NextRouter was not mounted. https://nextjs.org/docs/messages/next-router-not-mounted

### What browser are you using? (if relevant)

Error: NextRouter was not mounted. https://nextjs.org/docs/messages/next-router-not-mounted

### How are you deploying your application? (if relevant)

Error: NextRouter was not mounted. https://nextjs.org/docs/messages/next-router-not-mounted

### Describe the Bug

when i use useRouter 

### Expected Behavior

when i use useRouter 

### To Reproduce

when i use useRouter

**Comment by HamAndRock** [comment]
Without more info, It's really hard to help, but I assume that you are importing useRouter from `next/router` in app directory, in app directory you use be using useRouter imported from `next/navigation`.

Docs: https://beta.nextjs.org/docs/api-reference/use-router. 

Ideally please provide a repo where someone can reproduce the issue, if this doesn't fix your issue.

**Reply by icyJoseph** [reply]
This has to be the issue.

**Comment by BADHON512** [comment]
next/navigation   

wow it's work  thanks
